docs/testplan/Smartswitch-test-plan.md

```diff
@@ -645,6 +672,673 @@ root@sonic:/home/cisco# show chassis modules status
  *  Verify number of DPUs from inventory file for the testbed and number of DPUs shown in the cli output.
 
 
+### 1.13 Check Memory on DPU
+
+### Steps
+
+ * Infrastructure support will be provided to choose from following clis or in combinations based on the vendor.
+ * Use command `show system-memory` to get memory usage on each of those DPUs
+ * Use `show system-health dpu <DPU_NUM>` to check memory check service status
+ * Use `pdsctl show system --events` to check memory related events triggered.
+   (This is vendor specific event monitoring cli) 
```

Comment:
Does it mean that the command is specific to Cisco? or does it invoke vendor specific API?

Reply:
Yes last command is vendor specific. 
The objective is to provide general function which expects vendor specific cli and expected output. 
Then the function just can execute and compare it to provide the result

Reply:
Is so, can you capture NVIDIA specific command as well?

Reply:
I am not sure about Nvidia's command. 
But during the call, they agreed to it on first two steps itself. The third one I added it extra check from our side. 
If any one wants to add any extra checks, infra will be provided.